db-engine/pom.xml

```diff
@@ -237,6 +237,19 @@
                     </execution>
                 </executions>
             </plugin>
+             <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-surefire-plugin</artifactId>
+                        <version>2.19.1</version>
+                        <configuration>
+                            <additionalClasspathElements>
+                                <additionalClasspathElement>path/to/additional/resources</additionalClasspathElement>
```

Comment:
Yes unfortunately, I want my DataValueDescriptor tests to run in db-engine but since they are dependent on ClassSize codegen it is a bit more involved.